src/actions.js

```diff
@@ -607,3 +400,41 @@ export const updateSyncWarnings: UpdateSyncWarnings = (
   meta: { form },
   payload: { syncWarnings, warning }
 })
+
+const actions = {
+  arrayInsert,
+  arrayMove,
+  arrayPop,
+  arrayPush,
+  arrayRemove,
+  arrayRemoveAll,
+  arrayShift,
+  arraySplice,
+  arraySwap,
+  arrayUnshift,
+  autofill,
+  blur,
+  change,
+  clearSubmit,
+  clearSubmitErrors,
+  clearAsyncError,
+  destroy,
+  focus,
+  initialize,
+  registerField,
+  reset,
+  startAsyncValidation,
+  startSubmit,
+  stopAsyncValidation,
+  stopSubmit,
+  submit,
+  setSubmitFailed,
+  setSubmitSucceeded,
+  touch,
+  unregisterField,
+  untouch,
+  updateSyncErrors,
+  updateSyncWarnings
+}
+
+export default actions
```

Comment:
@erikras I think you missed out documenting this breaking change in v7. No longer `import { actionName } from '../actions';`, but `import actions from '../actions'; const { actionName } = actions`. Not sure if it's possible to edit the v7 release notes